_includes/language.html

```diff
@@ -0,0 +1,17 @@
+<select id="language-select">
+  {% for language in site.languages %}
+    {% if language == site.lang %}
+      <option value="/" selected >
+        {{ language }}
+      </option>  
+    {% elsif language == 'en' %}
+      <option value="{{ site.baseurl_root | append: page.url }}">
+        {{ language }}
+      </option>
+    {% else %}
+      <option value="{{ page.url | prepend: language | prepend: '/' }}">
```

Comment:
I took out that baseurl, they should all be relative now and they still seem to work.